docs/cli/commands/run.md

```diff
@@ -22,5 +22,10 @@ npx cap run [options] <platform>
 - `--list`: Print a list of target devices available to the given platform
 - `--no-sync`: do not run the sync command
 - `--scheme <schemeName>`: set the scheme of the iOS project
+- `--configuration <name>`: Configuration name of the iOS Scheme
 - `--target <id>`: Run on a specific target device
+- `--live-reload`: Enable Live Reload
+- `--host <host>`: Live Reload by serving the web view on the specified host
+- `--port <port>`: Live Reload by serving the web view on the specified port
```

Comment:
The descriptions are a bit confusing, Capacitor doesn't actually serve the app for live reload, the user is supposed to have a server that provides live reload features and start it, and then use `--live-reload` to make capacitor load from the host specified in `--host` and the port specified in `--port` and those values should match the live reload server host/port, but the description seems to indicate that Capacitor will be the one in charge of serving.

Reply:
I've updated this. Hopefully its a bit clearer.